### ISSUE TITLE
Cannot convert object with single quotes in string

### ISSUE THREAD

**Issue by uhyo** [nd-02110114/babel-plugin-object-to-json-parse#11]
This plugin fails to convert following object literal:

```js
const obj = {
  foo: "'abc'"
};

console.log(obj.foo);
```

**result:**

```js
At 1 line (start) : The object wasn't converted (Unexpected token, expected "," (1:22) - make sure this is an expression.
> 1 | (JSON.parse('{"foo":"'abc'"}'))
    |             
```

**Comment by nd-02110114** [comment]
Maybe, I fixed this issue by #13. 
Please confirm 🙇

**Comment by uhyo** [comment]
Nice fix, thank you!
